scripts/Package.hs

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env stack
+-- stack --resolver lts-12.12 script --package Cabal,directory,extra,filepath,Glob,process,temporary
+
+{-# LANGUAGE LambdaCase #-}
+
+import Control.Monad
+import Control.Monad.Extra
+import Data.Maybe
+import Data.Monoid
+import Distribution.Simple.Program.Find
+import Distribution.Simple.Utils
+import Distribution.Verbosity
+import System.Directory
+import System.Environment
+import System.FilePath
+import System.Process
+import System.FilePath.Glob
+import System.IO.Temp
+
+packageBaseUrl = "https://s3-us-west-2.amazonaws.com/packages-luna/dataframes/windows-package-base.7z"
```

Comment:
I think it should be split into https://s3-us-west-2.amazonaws.com/packages-luna/dataframes and file name, as the s3 url is used also in line 75

Reply:
Huh. I really like being able to copy/paste URLs and the fact that they are both hosted in the same path is not inherent to them. I hope to get rid of the package skeleton soon anyway, so for now I'd just leave it as is.

Reply:
FWIW, you can substitute `"https://s3-us-west-2.amazonaws.com/packages-luna"` with `"https://packages.luna-lang.org"` -- then it will go through the CDN of ours. Plus, the URLs look nicer ;)

Reply:
👍

Reply:
You can use following syntax:

```
depsArchiveUrl, packageBaseUrl :: String
depsArchiveUrl = "https://s3-us-west-2.amazonaws.com/packages-luna/dataframes/libs-dev-v140.7z"
packageBaseUrl = "https://s3-us-west-2.amazonaws.com/packages-luna/dataframes/windows-package-base.7z"
```

Reply:
Oh, interesting, I didn't know this one.